tests/jump_spec.lua

```diff
@@ -662,6 +662,54 @@ describe('Jumping with f/t/F/T', function()
     validate_disable('g')
     validate_disable('b')
   end)
+
+  it('respects vim.o.ignorecase', function()
+    child.cmd('set ignorecase')
+    set_lines({ ' 1e2E3E4e_ ' })
+
+    set_cursor(1, 0)
+    type_keys('2f', 'E')
+    eq(get_cursor(), { 1, 4 })
+
+    set_cursor(1, 0)
+    type_keys('2t', 'E')
+    eq(get_cursor(), { 1, 3 })
+
+    set_cursor(1, 10)
+    type_keys('2F', 'E')
+    eq(get_cursor(), { 1, 6 })
+
+    set_cursor(1, 10)
+    type_keys('2T', 'E')
+    eq(get_cursor(), { 1, 7 })
+
+    child.cmd('set noignorecase')
```

Comment:
For possible future: no need to unset anything, as child process is restarted before every case here.